modules/configuring-the-quickstarts/proc-customize-rhdh-quickstart.adoc

```diff
@@ -0,0 +1,55 @@
+[id="customizing-your-product-quickstart_{context}"]
+= Customizing your {product} Quickstart
+
+You can configure the {product} Quickstart plugin to create a customized onboarding experience for your {product-short} administrators. 
+
+.Prerequisites
+You must have sufficient permissions as a platform engineer.
```

Comment:
If permissions are enabled then you must be an admin to be able to access the quickstart.